src/components/issues/Card.tsx

```diff
@@ -136,7 +136,16 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
                     </button>
                 )}
                 {isUserAuthorized && taskExists && devMode && (
-                    <ActionForm taskId={taskId || ''} />
+                    <ActionForm
+                        taskId={taskId || ''}
+                        createTask={async () => {
+                            console.log('Placeholder');
+                        }}
+                        updateTask={async () => {
+                            console.log('Placeholder');
+                        }}
+                        taskAssignee=""
+                    />
```

Comment:
This is fixed in this PR: [958](https://github.com/Real-Dev-Squad/website-status/pull/958/files#diff-3911623737bbd829be31d7d883733f7e8b1045f1fe60a33366a5132f581c24acR284-R288)